TLSharp.Core/TelegramClient.cs

```diff
@@ -377,8 +377,15 @@ private void OnUserAuthenticated(TLUser TLUser)
             _session.SessionExpires = int.MaxValue;
 
             _session.Save();
-        }
-
+        }
+
+        public bool IsConnected()

```

Comment:
@salarcode I've fixed it myself and merged manually, feel free to propose the KeepAlive PR now ;)